src/rpcmining.cpp

```diff
@@ -381,10 +381,26 @@ UniValue getblocktemplate(const UniValue& params, bool fHelp)
             "  \"curtime\" : ttt,                  (numeric) current timestamp in seconds since epoch (Jan 1 1970 GMT)\n"
             "  \"bits\" : \"xxx\",                 (string) compressed target of next block\n"
             "  \"height\" : n                      (numeric) The height of the next block\n"
-            "  \"payee\" : \"xxx\",                (string) required payee for the next block\n"
-            "  \"payee_amount\" : n,               (numeric) required amount to pay\n"
-            "  \"masternode_payments\" : true|false,         (boolean) true, if masternode payments are enabled\n"
-            "  \"enforce_masternode_payments\" : true|false  (boolean) true, if masternode payments are enforced\n"
+            "  \"masternode\" : [                  (array) required masternode payee that must be included in the next block\n"
+            "      {\n"
+            "         \"payee\" : \"xxxx\",          (string) payee address\n"
+            "         \"script\" : \"xxxx\",         (string) payee scriptPubKey\n"
+            "         \"amount\": n                (numeric) required amount to pay\n"
+            "      }\n"
+            "      ,...\n"
+            "  ],\n"
```

Comment:
compiled locally and testing, found that getblocktemplate result is
```  "masternode": {
    "payee": "yUuAsYCnG5XrjgsGvRwcDqPhgLUnzNfe8L",
    "script": "76a9145e21cfe3b3ce2e6027bff8c73db872a89561871188ac",
    "amount": 1250000000
  },
```
Are we having more than one masternode payment object in a block ? (for `,...`)